src/component/playground/PlaygroundResultsTable/PlaygroundResultsTable.tsx

```diff
@@ -0,0 +1,187 @@
+import { PageContent } from 'component/common/PageContent/PageContent';
+import { PageHeader } from 'component/common/PageHeader/PageHeader';
+import {
+    SortableTableHeader,
+    Table,
+    TableBody,
+    TableCell,
+    TablePlaceholder,
+    TableRow,
+} from 'component/common/Table';
+import { useGlobalFilter, useSortBy, useTable } from 'react-table';
+import { SearchHighlightProvider } from 'component/common/Table/SearchHighlightContext/SearchHighlightContext';
+import { sortTypes } from 'utils/sortTypes';
+import { useMemo, useState } from 'react';
+import { HighlightCell } from 'component/common/Table/cells/HighlightCell/HighlightCell';
+import { ConditionallyRender } from 'component/common/ConditionallyRender/ConditionallyRender';
+import { Search } from 'component/common/Search/Search';
+import { usePlaygroundResults } from '../../../hooks/api/getters/usePlaygroundResults/usePlaygroundResults';
+import { LinkCell } from '../../common/Table/cells/LinkCell/LinkCell';
+import { FeatureStatusCell } from '../StatusCell/FeatureStatusCell';
+import { useSearchParams } from 'react-router-dom';
+import { useSearch } from '../../../hooks/useSearch';
+import { ContextBanner } from '../ContextBanner/ContextBanner';
+
+export const PlaygroundResultsTable = () => {
+    const { features, input, loading } = usePlaygroundResults();
+    const { context } = input;
+    const [initialState] = useState({
```

Comment:
If we are never changing this, it shouldn't be in state. State should only keep a record of data changing over time, anything that doesn't change or can be computed should be outside of state.